ct/actualbudget.sh

```diff
@@ -26,10 +26,89 @@ function update_script() {
 
     if [[ ! -d /opt/actualbudget ]]; then
         msg_error "No ${APP} Installation Found!"
-        exit 1
+        exit
     fi
-    msg_error "Due to major changes in the Actual Budget repository, we are currently unable to provide updates. Please check back later."
-    exit
+
+    RELEASE=$(curl -s https://api.github.com/repos/actualbudget/actual/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+    if [[ ! -f /opt/actualbudget_version.txt ]] || [[ "${RELEASE}" != "$(cat /opt/actualbudget_version.txt)" ]]; then
+        msg_info "Stopping ${APP}"
+        systemctl stop actualbudget
+        msg_ok "${APP} Stopped"
+
+        msg_info "Updating ${APP} to ${RELEASE}"
+        cd /tmp
+        wget -q https://github.com/actualbudget/actual/archive/refs/tags/v${RELEASE}.tar.gz
+
+        mv /opt/actualbudget /opt/actualbudget_bak
+        $STD tar -xzf "v${RELEASE}.tar.gz"
```

Comment:
```suggestion
        tar -xzf "v${RELEASE}.tar.gz"
```

$STD should not be needed as no -v flag is passed to tar